tests/foreman/ui/test_settings.py

```diff
@@ -209,53 +211,58 @@ def test_positive_selectors(session):
                 setting_cleanup(setting_attr, str(original_val.default))
 
 
-@stubbed()
 @tier3
-def test_positive_update_login_page_footer_text_with_long_string():
-    """Attempt to update parameter "Login_page_footer_text"
-        with long length string under General tab
+def test_positive_update_login_page_footer_text_with_long_string(session):
+    """Testing to update parameter "Login_page_footer_text with long length
+    string under General tab
 
     :id: b1a51594-43e6-49d8-918b-9bc306f3a1a2
 
-    :steps:
-
-        1. Navigate to Administer -> settings
-        2. Click on general tab
-        3. Input long length string into login page footer field
-        4. Assert value from login page
-
     :expectedresults: Parameter is updated
 
     :CaseImportance: Medium
 
-    :CaseAutomation: notautomated
-
     :CaseLevel: Acceptance
     """
+    property_name = 'login_text'
+    property_value = [
+        ele.value
+        for ele in entities.Setting().search(
+            query={'search': 'name={}'.format(property_name)}
+        )
+    ][0]
+    login_text_data = gen_string('alpha', 270)
+    with session:
+        try:
+            session.settings.update("name={}".format(property_name),
+                                    "{}".format(login_text_data))
+            result = session.settings.read("name={}".format(property_name))
+            assert result["table"][0]["Value"] == login_text_data
```

Comment:
Can you verify this on login page, you can use the logout method as well to navigate there.

Reply:
Yeah, I think we can skip checking if setting is updated, we are much more interested in footer on login page.

The problematic part is that while we expose `logout()` method of base view, it actually doesn't do anything. See https://github.com/SatelliteQE/airgun/pull/397 and discussion in it. I think logout should be modeled after `RHSSOLoginEntity::logout`, i.e. after logging out it should instantiate login page view and return `read()` on that.

Another problem is that our login view exposes very limited set of widgets, and footer is not one of them.

Reply:
@mirekdlugosz RHSSOLoginEntity:: logout is special logout which navigates to intermediate page. We will need to merge SatelliteQE/airgun#397. This is waiting for a long time, we can avoid the to get used by all other test cases. @mirekdlugosz what's your view?

Reply:
> Yeah, I think we can skip checking if setting is updated, we are much more interested in footer on login page.
Agree.
> 
> The problematic part is that while we expose `logout()` method of base view, it actually doesn't do anything. See [SatelliteQE/airgun#397](https://github.com/SatelliteQE/airgun/pull/397) and discussion in it. I think logout should be modeled after `RHSSOLoginEntity::logout`, i.e. after logging out it should instantiate login page view and return `read()` on that.
> 
> Another problem is that our login view exposes very limited set of widgets, and footer is not one of them.

Yes, it is still in review state, once it gets merge then we will have two options to achieve it.

1- Raise the PR for view update(add the login_text locator in RhssoExternalLogoutView) and Use RHSSOLoginEntity:: logout

2- Update the omkar's logout PR.

I tested it in both ways but the login page is not properly loaded, Hence, We are unable to read the login view's content. It is possible if we modify the element mentioned in browser_wait_for_element and that is not the correct way. 
Can anybody tell me how we can wait for a few seconds in LoginEntity::logout/RHSSOLoginEntity::logout method before calling their login view?

Reply:
> I tested it in both ways but the login page is not properly loaded, Hence, We are unable to read the login view's content. It is possible if we modify the element mentioned in browser_wait_for_element and that is not the correct way.
> Can anybody tell me how we can wait for a few seconds in LoginEntity::logout/RHSSOLoginEntity::logout method before calling their login view?

Problem resolved.

Reply:
Raised Airgun PR https://github.com/SatelliteQE/airgun/pull/474